protocol_test.go

```diff
@@ -4,34 +4,42 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	"github.com/tarantool/go-iproto"
 
 	. "github.com/tarantool/go-tarantool/v2"
 )
 
 func TestProtocolInfoClonePreservesFeatures(t *testing.T) {
 	original := ProtocolInfo{
 		Version:  ProtocolVersion(100),
-		Features: []ProtocolFeature{ProtocolFeature(99), ProtocolFeature(100)},
+		Features: []iproto.Feature{iproto.Feature(99), iproto.Feature(100)},
 	}
 
 	origCopy := original.Clone()
 
-	original.Features[1] = ProtocolFeature(98)
+	original.Features[1] = iproto.Feature(98)
 
 	require.Equal(t,
 		origCopy,
 		ProtocolInfo{
 			Version:  ProtocolVersion(100),
-			Features: []ProtocolFeature{ProtocolFeature(99), ProtocolFeature(100)},
+			Features: []iproto.Feature{iproto.Feature(99), iproto.Feature(100)},
 		})
 }
 
 func TestFeatureStringRepresentation(t *testing.T) {
-	require.Equal(t, StreamsFeature.String(), "StreamsFeature")
-	require.Equal(t, TransactionsFeature.String(), "TransactionsFeature")
-	require.Equal(t, ErrorExtensionFeature.String(), "ErrorExtensionFeature")
-	require.Equal(t, WatchersFeature.String(), "WatchersFeature")
-	require.Equal(t, PaginationFeature.String(), "PaginationFeature")
-
-	require.Equal(t, ProtocolFeature(15532).String(), "Unknown feature (code 15532)")
+	require.Equal(t, iproto.IPROTO_FEATURE_STREAMS.String(),
+		"IPROTO_FEATURE_STREAMS")
+	require.Equal(t, iproto.IPROTO_FEATURE_TRANSACTIONS.String(),
+		"IPROTO_FEATURE_TRANSACTIONS")
+	require.Equal(t, iproto.IPROTO_FEATURE_ERROR_EXTENSION.String(),
+		"IPROTO_FEATURE_ERROR_EXTENSION")
+	require.Equal(t, iproto.IPROTO_FEATURE_WATCHERS.String(),
+		"IPROTO_FEATURE_WATCHERS")
+	require.Equal(t, iproto.IPROTO_FEATURE_PAGINATION.String(),
+		"IPROTO_FEATURE_PAGINATION")
+	require.Equal(t, iproto.IPROTO_FEATURE_WATCH_ONCE.String(),
+		"IPROTO_FEATURE_WATCH_ONCE")
+
+	require.Equal(t, iproto.Feature(15532).String(), "Feature(15532)")
 }
```

Comment:
We could remove the tests since it is an external type.

Reply:
Updated.